build.sbt

```diff
@@ -6,7 +6,7 @@ organization in Global := "edu.gemini.ocs"
 
 ocsVersion in ThisBuild := OcsVersion("2016A", true, 1, 1, 1)
 
-pitVersion in ThisBuild := OcsVersion("2016A", false, 1, 1, 0)
+pitVersion in ThisBuild := OcsVersion("2016B", true, 2, 1, 0)
```

Comment:
Right, the original problem is that OSGi doesn't support letters on the version number and I wanted to have an easy way to tell the semesters apart. So to make OSGi happy we made the bundle version 2016.2.1 which is easy to parse by humans too